Library/Homebrew/test/cask/cask_spec.rb

```diff
@@ -46,13 +46,15 @@
       expect(c.token).to eq("caffeine")
     end
 
-    it "returns an instance of the Cask from a URL" do
+    it "returns an instance of the Cask from a URL", :allow_utils_curl do
+      expect(Homebrew::API).not_to receive(:fetch_json_api_file)
```

Comment:
This is just a sanity check to prevent problems in the future.

Reply:
Can you clarify what exactly the problem here would be? Allowing `curl` but still preventing the API from being fetched?

Can this be a separate `:no_api` spec helper, given it's used three times?

Reply:
Addressed in https://github.com/Homebrew/brew/pull/16903/commits/74aea8e92d78201db65b6eedfd9ec60954264fc1. Just setting `HOMEBREW_NO_INSTALL_FROM_API` should be good enough here.